lib/stages/FEngine.cpp

```diff
@@ -0,0 +1,545 @@
+#include <Config.hpp>
+#include <Stage.hpp>
+#include <StageFactory.hpp>
+#include <cassert>
+#include <chimeMetadata.hpp>
+#include <chordMetadata.hpp>
+#include <complex>
+#include <cstdint>
+#include <cstring>
+#include <fstream>
+#include <julia.h>
+#include <juliaManager.hpp>
+#include <string>
+#include <vector>
+
+// Find float16:
```

Comment:
There is also some Float16 handling in lib/util/visUtil.hpp -- consider moving this code to there and including visUtil.hpp ?

Reply:
Did that.